packages/edit-widgets/src/index.js

```diff
@@ -37,7 +37,7 @@ import {
 const disabledBlocks = [
 	'core/more',
 	'core/freeform',
-	...( ! ALLOW_REUSABLE_BLOCKS && [ 'core/block' ] ),
+	...( ! ALLOW_REUSABLE_BLOCKS ? [ 'core/block' ] : [] ),
```

Comment:
Nit: Would be slightly easier to read if we negate the ternary.

```suggestion
	...( ALLOW_REUSABLE_BLOCKS ? [] : [ 'core/block' ] ),
```

Reply:
Probably. I have a weird thing where I don't like having the "empty" bit first but 🤷 

Perhaps changing the _constant_ to `DISABLE_RESUABLE_BLOCKS` would make this whole thing make more sense?

Reply:
I don't mind either way 😆.

Reply:
>  I have a weird thing where I don't like having the "empty" bit first but 🤷

I do that all the time as well 😆